scipy/optimize/tests/test_linprog.py

```diff
@@ -1592,15 +1592,12 @@ def setup_method(self):
         self.options = {}
 
     def test_bug_5400(self):
-        with pytest.raises(ValueError):
-            super(TestLinprogSimplexDefault, self).test_bug_5400()
+        pytest.skip("Simplex fails on this problem.")
```

Comment:
No sense in replacing these tests. We basically do not support `simplex` any more.